### ISSUE TITLE
share link should have a 'copy to clipboard' button

### ISSUE THREAD

**Issue by robes** [informatics-isi-edu/chaise#1754]
This is pretty standard. Its a little confusing what to do with the resolver link. Its a clickable url but goes to the exact same page you are already on. What we really want is for user to copy it. We should have a copy-to-clipboard icon next to it. This seems like its pretty standard.

https://www.w3schools.com/howto/howto_js_copy_clipboard.asp

**Comment by jrchudy** [comment]
We decided to take a slightly different approach to resolve this issue than what the w3 link suggests we do. 
![image](https://user-images.githubusercontent.com/2932600/59532508-d8cf8880-8e9d-11e9-93d5-11657356f2a2.png)

What we have implemented is an icon (looks like a copy to clipboard icon), that when clicked, will copy the link to the clipboard. We didn't want to clutter the UX more with an input box for copying from or popping up another dialog on top of the modal dialog.
